### DISCUSSION TITLE
Desplegar info de Api en Html

### DISCUSSION THREAD

**Discussion by LuisFelipeCajero** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#505] [Q&A]
Ya me salen los movimientos de los pokemon en Console pero como lo desplego en la página para que los visualice el usuario? :( aiudaaa jaja

**Comment by GloriaCoralCerecedo** [comment]
No se si te pueda ayudar mostrándote mi repo

**Reply by LuisFelipeCajero** [reply]
Gracias Gloria, me fue de mucha ayuda :)

**Reply by JossPerezT** [reply]
Gracias Gloria, me ayudaste con la misma duda. ⚡🚀

**Comment by doguedogue** [comment]
https://web.microsoftstream.com/video/71be733f-2b72-4c9f-8712-83fc981a8b84?list=user&userId=96eeb66f-b984-4083-8ea6-74c34a07a34a

**Comment by carlos-gmx3** [comment]
accedes a esa información a consultando del conjunto de datos objetos y arreglos
Te dejo algunos ejemplos que he usado
document.getElementById('pokeName').innerHTML = pokeObj.species.name;
type1.innerHTML = pokeObj.types[0].type.name;
type2.innerHTML = pokeObj.types[1].type.name;
no se si esa era tu duda

**Comment by JosueDanieLB** [comment]
Hola, lo que yo hice en el JavaScript, es crear una variable usando "let" que tome el valor del ataque o del stat de la ruta de la API

 **let pokeAtaque = data.moves[0].move.name;**

*Puedes cambiar el "0" por otra variable que cambie de valor para estar cambiando (en este caso) de ataque si el pokemon tiene varios.

Despues, con este comando cambias el valor del texto en el HTML

**document.getElementById("nombreAtaque").innerHTML = pokeAtaque;**

"nombreAtaque" es el ID de la etiqueta del HTML donde se debe mostrar el nombre del ataque (en este caso), yo lo he usado con <p> y con los <h>.

Saludos
